tests/test_utils.py

```diff
@@ -449,20 +449,38 @@ def delete_config_from_filesystem(
     )
 
 
+# noinspection PyPep8Naming
 def get_snowflake_connection_url() -> str:
     """Get snowflake connection url from environment variables.
 
     Returns:
-        String of the snowflake connection url.
+        String of the snowflake connection URL.
     """
-    sfAccount = os.environ.get("SNOWFLAKE_ACCOUNT")
     sfUser = os.environ.get("SNOWFLAKE_USER")
     sfPswd = os.environ.get("SNOWFLAKE_PW")
+    sfAccount = os.environ.get("SNOWFLAKE_ACCOUNT")
     sfDatabase = os.environ.get("SNOWFLAKE_DATABASE")
     sfSchema = os.environ.get("SNOWFLAKE_SCHEMA")
     sfWarehouse = os.environ.get("SNOWFLAKE_WAREHOUSE")
+    sfRole = os.environ.get("SNOWFLAKE_ROLE") or "PUBLIC"
+
+    return f"snowflake://{sfUser}:{sfPswd}@{sfAccount}/{sfDatabase}/{sfSchema}?warehouse={sfWarehouse}&role={sfRole}"
+
+
+def get_redshift_connection_url() -> str:
+    """Get Amazon Redshift connection url from environment variables.
+
+    Returns:
+        String of the Amazon Redshift connection URL.
+    """
+    host = os.environ.get("REDSHIFT_HOST")
```

Comment:
Since this is used for testing, would default values be useful here (ie `os.environ.get("REDSHIFT_HOST", <default_value>)`? 
My thought is someone wants to test locally via a dockerized service (I don't know if one exists for redshift) that we provide then they wouldn't have to set these environment variables locally.
This isn't a big deal. In general I like minimizing the configuration necessary to run tests.